shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -47,8 +47,13 @@ extern NSNotificationName const FlutterSemanticsUpdateNotification;
  * Dart-related state and asynchronous tasks when navigating back and forth between a
  * FlutterViewController and other `UIViewController`s.
  */
+#ifdef __IPHONE_13_4
```

Comment:
You shouldn't need this.  The presence of UIPointerInteractionDelegate is determined by the SDK version used to compile the engine.  We don't support multiple versions of the SDK.  The thing that needs to be guarded is the use of the API at runtime.

Reply:
Oops, thought I'd caught them all. Will remove.